crates/apollo-smith/src/description.rs

```diff
@@ -26,7 +26,7 @@ impl From<Description> for String {
 #[cfg(feature = "parser-impl")]
 impl From<apollo_parser::ast::Description> for Description {
     fn from(desc: apollo_parser::ast::Description) -> Self {
-        Description(StringValue::from(desc.to_string()))
+        Description(desc.string_value().map(|s| s.into()).unwrap_or_else(|| StringValue::Line(Default::default())))
```

Comment:
ah, yea the original line should have been `desc.text().to_string()`, since `.text()` strips away the extra quotation marks around string values. So that's something to keep in mind with this PR - source_string() for string values that have quotes around them should have those quotes stripped.